acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/site/ps/PublicationPage.java

```diff
@@ -4,70 +4,108 @@
 import uk.nhs.digital.ps.test.acceptance.webdriver.WebDriverProvider;
 import org.openqa.selenium.*;
+
+import java.util.Collection;
 import java.util.List;
 
+import static java.util.Arrays.asList;
 import static java.util.stream.Collectors.toList;
+import static uk.nhs.digital.ps.test.acceptance.pages.site.ps.PublicationPageElements.FieldKeys.*;
 
 public class PublicationPage extends AbstractSitePage {
 
     private PageHelper helper;
-    private PageElements pageElements;
+    private List<PageElements> pageElements;
 
     public PublicationPage(WebDriverProvider webDriverProvider, final PageHelper helper) {
         super(webDriverProvider);
         this.helper = helper;
-        this.pageElements = new SeriesPageElements();
+        this.pageElements = asList(new SeriesPageElements(), new PublicationPageElements());
     }
 
     public void open(final Publication publication) {
         getWebDriver().get(URL + "/publications/" + publication.getPublicationUrlName());
     }
 
     public String getTitleText() {
-        return pageElements.getElementByName("Publication Title", getWebDriver()).getText();
+        return findPageElement(PUBLICATION_TITLE).getText();
     }
 
     public String getSummaryText() {
-        return pageElements.getElementByName("Publication Summary", getWebDriver()).getText();
+        return findPageElement(SUMMARY).getText();
     }
 
-    public List<AttachmentElement> getAttachments() {
+    public List<AttachmentWidget> getAttachments() {
         return getWebDriver().findElements(By.cssSelector("li[class~='attachment']")).stream()
-            .map(webElement -> new AttachmentElement(getWebDriver(), webElement))
+            .map(webElement -> new AttachmentWidget(getWebDriver(), webElement))
             .collect(toList());
     }
 
-    public AttachmentElement findAttachmentElementByName(final String fullName) {
+    public AttachmentWidget findAttachmentElementByName(final String fullName) {
         return getAttachments().stream()
-            .filter(attachmentElement -> fullName.equals(attachmentElement.getFullName()))
+            .filter(attachmentWidget -> fullName.equals(attachmentWidget.getFullName()))
             .findFirst()
             .orElse(null);
     }
 
     public String getGeographicCoverage() {
-        return pageElements
-            .getElementByName("Publication Geographic Coverage", getWebDriver())
-            .getText();
+        return findPageElement(GEOGRAPHIC_COVERAGE).getText();
     }
 
     public String getInformationType() {
         return helper.findElement(By.xpath("//*[@data-uipath='ps.publication.information-types']")).getText();
     }
 
     public String getGranularity() {
-        return pageElements
-            .getElementByName("Publication Granularity", getWebDriver())
-            .getText();
+        return findPageElement(PublicationPageElements.FieldKeys.GRANULARITY).getText();
```

Comment:
You can lose the `PublicationPageElements.FieldKeys.` qualifier thanks to your static imports, I think, here and elsewhere in this file?

Reply:
Done.